server/routes/risk.ts

```diff
@@ -0,0 +1,56 @@
+import { LicenceService } from '../services/licenceService'
+import type { Response } from 'express'
+import type { LicenceLocals } from '../utils/middleware'
+import { asyncMiddleware } from '../utils/middleware'
+import createStandardRoutes from './routeWorkers/standard'
+import formConfig from './config/risk'
+import { getIn, firstItem } from '../utils/functionalHelpers'
+
+const sectionName = 'risk'
+
+export default ({ licenceService }: { licenceService: LicenceService }) =>
+  (router, audited) => {
+    const standard = createStandardRoutes({ formConfig, licenceService, sectionName })
+
+    router.get('/:formName/:bookingId', asyncMiddleware(getStandard))
+
+    function getStandard(req, res: Response<any, LicenceLocals>) {
+      const { formName, bookingId, action } = req.params
+      const { licenceSection, nextPath, pageDataMap } = formConfig[formName]
+      const dataPath = pageDataMap || ['licence', sectionName, licenceSection]
+      const riskVersion = licenceService.getRiskVersion(res.locals.licence.licence)
+
+      const rawData = getIn(res.locals.licence, dataPath) || {}
+      const data =
+        firstItem(req.flash('userInput')) || licenceService.addSplitDateFields(rawData, formConfig[formName].fields)
+      const errorObject = firstItem(req.flash('errors')) || {}
+
+      const viewData = { bookingId, data, nextPath, errorObject, action, sectionName, formName, version: riskVersion }
```

Comment:
we can hardcodee sectionName/formName though I'm surprised they are used.. We maybe able to reduce this viewData to only that which is required by this screen